content/en/gitops.md

```diff
@@ -1,29 +1,23 @@
 ---
 title: GitOps
-status: Feedback Appreciated
+status: Completed
 category: Concept
 tags: ["methodology", "", ""]
 ---
 
-GitOps is a set of best practices based on [shared principles](https://opengitops.dev/), 
-applied to a workflow that depends on software agents that 
-enable automation to reconcile a declared system state or configuration in a git repository.
-These software agents and practices are used to execute a cohesive workflow that 
-leverages a source control system like Git as the “single source of truth” and 
-extends this practice to applications, infrastructure, and operational procedures.
+GitOps is a set of practices for managing software applications and infrastructure by continuously evaluating and reconciling their desired states as defined in a version control system such as Git against their actual state. Reconciling states includes changing the resources to match the defined state via deployment or updates.
+
+In GitOps, the state or configuration of a system is fully described in files stored in the source of truth, usually the version control system. It could, however, be another source of truth, such as a Kubernetes Operator or Artifact Storage.
```

Comment:
```suggestion
In GitOps, the state or configuration of a system is fully described in files stored in the source of truth, usually the version control system. 
It could, however, be another source of truth, such as a Kubernetes Operator or Artifact Storage.
```